src/PaymentSuite/FreePaymentBundle/Resources/config/controllers.yml

```diff
@@ -5,6 +5,7 @@ services:
     #
     paymentsuite.freepayment.payment_controller:
         class: PaymentSuite\FreePaymentBundle\Controller\PaymentController
+        public: true
```

Comment:
why do you need this @gricob?

Reply:
Using controller service in routes requires it to be public. Symfony 4 make services private by default, so the service sould be declared as public explicitilly

Reply:
do you mean this? https://github.com/PaymentSuite/paymentsuite/blob/ba87aaa92e4d4e2dd97f5a441bc0bd05f67b9016/src/PaymentSuite/PaylandsBundle/Resources/config/routing.yml#L4

Reply:
Yes

Reply:
Good to know, and something to care about it, because I think it has to be another way to allow override controllers and keep the route but without make public the controller. what do you think @sangarbe ?